src/content/core/components/namespaces.mdx

```diff
@@ -31,18 +46,27 @@ To share a namespace go to the Okteto dashboard, select the namespace you want t
 
 <p align="center"><Image src={require('@site/static/img/share.png').default} alt="share a namespace" width="850" /></p>
 
-In the `Share` dialog, type the email address of the team members you want to share this namespace with. Once you press save, Okteto dashboardll update the necessary permissions and notify your teammates via email.
+In the `Share` dialog, type the email address of the team members you want to share this namespace with. Once you press save, Okteto dashboard will update the necessary permissions and notify your teammates via email.
 
-You can sleep your namespace from the command line with `okteto namespace sleep [name]`.
 
-When you want to wake your namespace, you can use the command `okteto namespace wake [name]`.
+## Transfer your namespace
+
+Okteto administrators have the ability to transfer non-default namespaces between users. This process can be achieved through the Okteto dashboard.
+
+To transfer a namespace to a new owner, go to the Okteto dashboard -> Admin -> Namespaces then locate the namespace that you would like to transfer. Click on the three dots `...` to display the namespace menu. From the namespace menu select Change Owner.
 
-You can sleep your namespace from Okteto dashboardself by clicking on Sleep dialog.
+<p align="center"><Image src={require('@site/static/img/namespace-change-owner.png').default} alt="share a namespace" width="650" /></p>
+
+When the Change Owner modal appears, type the email address of the team member who you'd like the transfer the namespace to. Once you press confirm, ownership of that namespace will be immediately transferred.
+If you wish, you can keep the current owner as a member of the namespace by selecting the option in the Change Owner modal. If this is not selected the current owner will immediately lose access.
+
+<p align="center"><Image src={require('@site/static/img/namespace-transfer-modal.png').default} alt="share a namespace" width="700" /></p>
 
-In order to wake the namespace, you can click on Wake dialog.
 
 :::tip
 
-SaaS single tenant and Self-Hosted customers can specify a global, default threshold for automatically sleeping a namespace in the [garbage collection](reference/helm-chart-values.mdx/#gc) configuration.
+User specific secrets will not transfer along with a namespace. If the namespace being transferred has user specific secrets, please note that future build or destroy actions may fail.
```

Comment:
```suggestion
User specific variables will not transfer along with a namespace. If the namespace being transferred has user specific variables, please note that future build or destroy actions may fail.
```

Reply:
> If the namespace being transferred has user specific variables

The namespace doesn't have variables. The users are the one having variables and then, they can be used in deployed dev environments within that namespace.

So, the right sentence would be something like: `If previous owner had some user variables used to deploy some active dev environment, please note that future build, redeploy or destroy actions may fail`